docker/entrypoint.sh

```diff
@@ -81,7 +81,7 @@ case ${entry_cmd} in
 	        	echo "Skipping: ${test_py}"
```

Comment:
Also skip the SensorThingsAPI tests (needs local server). See #1093 latest version of this file.

Reply:
Thanks! Fixed as well.